src/units/abilities.cpp

```diff
@@ -1963,6 +1965,15 @@ effect::effect(const unit_ability_list& list, int def, const_attack_ptr att, boo
 			}
 		}
 
+		if(type == "resistance"){
+			if(cfg.has_attribute("max_value")){
+				max_value = max_value ? std::max(*max_value, cfg["max_value"].to_int()) : cfg["max_value"].to_int();
+			}
+			if(cfg.has_attribute("min_value")){
+				min_value = min_value ? std::min(*min_value, cfg["min_value"].to_int()) : cfg["min_value"].to_int();
```

Comment:
If multiple abilities specify a minimum, then I'd expect the narrowest bounds to apply. So the std::max of the min_values and the std::min of the max_values.

Reply:
Ok but it changes behavior in resistance here it is highest value of max_value who is choose.

Reply:
I think that's the right behavior, but I'm leaving the conversation unresolved for Pentarctagon to read.

Reply:
It makes sense to me.